src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -294,54 +292,31 @@ private void executeWorker(Webhook webhook, Card card) {
     }
 
     private void addScmDetails() {
+        Set<User> users;
+        List<ChangeLogSet<ChangeLogSet.Entry>> sets;
+
         try {
-            if (run instanceof AbstractBuild) {
-                AbstractBuild build = (AbstractBuild) run;
-                factsBuilder.addCulprits(run.getResult(), build.getCulprits());
-
-                ChangeLogSet changeSet = build.getChangeSet();
-                Set<ChangeLogSet.AffectedFile> files = new HashSet<>();
-                Set<User> authors = new HashSet<>();
-
-                for (Object o : changeSet.getItems()) {
-                    ChangeLogSet.Entry entry = (ChangeLogSet.Entry) o;
-                    authors.add(entry.getAuthor());
-                    files.addAll(getAffectedFiles(entry));
-                }
+            users = (Set<User>) run.getClass().getMethod("getCulprits").invoke(run);
```

Comment:
Nice cleanup, huh? 👍
I tested against baseline 1.651.2 and 2.73.2

For reference:
https://github.com/jenkinsci/token-macro-plugin/blob/acea69fd410e7b36dda6a72ab596686f250edea3/src/main/java/org/jenkinsci/plugins/tokenmacro/impl/ChangesSinceLastBuildMacro.java#L100-L101

With even older core baseline 👍 
https://github.com/jenkinsci/token-macro-plugin/blob/4d24aa5716f8d84e236824ab36d87c90a9897e8d/pom.xml#L12